s3config.release.yaml.sample

```diff
@@ -19,6 +19,8 @@
 
 S3Config_Sections: [S3_SERVER_CONFIG, S3_AUTH_CONFIG, S3_MOTR_CONFIG, S3_THIRDPARTY_CONFIG, S3_VERSION_CONFIG]
 S3_SERVER_CONFIG:                                       # Section for S3 Server
+   S3_DI_DISABLE_DATA_CORRUPTION_IEM: false             # disable IEM in case of data integrity checksum fails
```

Comment:
disable sending of IEM messages by s3server in case ....

Reply:
fixed